back/.env

```
@@ -11,3 +11,5 @@ DATABASE_PORT=5432
 DATABASE_USERNAME="postgres"
 DATABASE_PASSWORD="1234"
 DATABASE_NAME="transcendence"
+
+WEBSOCKET_CORS_HOST="http://127.0.0.1:5500"
```

```suggestion
WEBSOCKET_CORS_HOST="http://localhost:3000"
```

기본값을 dockerfile로 배포시 설정되는 프론트 주소값으로 해주세요

---

넵 수정했습니다